File: isSubString.py
def isSubString(): 
	input1 = raw_input("Please enter a string: ")
	input2 = raw_input("Please enter another: ")
	if(len(input1) >= len(input2)):
		stringA = input2
		stringB = input1
	else:
		stringA = input1
		stringB = input2
	for i in range(0,((len(stringB) - len(stringA)) + 1)):
		if(stringB[i:i + len(stringA)] == stringA):
			return True
	return False

File: isSubString.cpp
//check if string is a substring c++
#include <iostream>
#include <string>
using namespace std;

bool isSubString()
{
	string input1;
	string input2;
	string stringA;
	string stringB;
	cout << "Please input a string\n";
	getline(cin, input1);
	cout << "Please input another string\n";
	getline(cin, input2);
	if (input1.size() >= input2.size())
	{
		stringA = input2;
		stringB = input1;
	}
	else
	{
		stringB = input2;
		stringA = input1;
	}
	return true;
}
